tensorflow/core/kernels/conv_grad_input_ops.cc

```diff
@@ -341,15 +341,9 @@ void LaunchConv2DBackpropInputOp<GPUDevice, T>::operator()(
       AsDeviceMemory(pre_transformed_in_backprop.template flat<T>().data(),
                      pre_transformed_in_backprop.template flat<T>().size());
 
-  int64_t workspace_bytes = 1LL << 32;  // 4GB by default.
-  // CuDNN frontend will expose more engines some of which might use too much
```

Comment:
So we are no longer restricting it to 1GB in the case listed here. Has this been addressed in cudnn/no longer concern? (I don't know if there is a test that corresponds to this that would capture the degenerate behavior here).

Reply:
Yes, this restriction was added when we introduced the cudnn frontend APIs and we encountered some failed unit tests. This type of failure is hard to capture since it depends on the heuristics returned from cudnn, autotuning, and tests (where multiple dgrad conv ops can be issued in parallel).